1-hello-world/.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
+# using gcloud. It follows the same syntax as .gitignore, with the addition of
+# "#!include" directives (which insert the entries of the given .gitignore-style
+# file at that point).
+#
+# For more information, run:
+#   $ gcloud topic gcloudignore
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Node.js dependencies:
+node_modules/
```

Comment:
I am not sure we should version this file. It is automatically generated at deployment time if not present.

Reply:
Yeah I could really go either way 🤷‍♂️  Your call :)

Reply:
Leaving this for @fhinkel to choose